resources/views/committee/members.blade.php

```diff
@@ -10,8 +10,20 @@
 
             <div class="panel-body">
                 <p>
+                    <b><u>Columns</u></b><br>
```

Comment:
You generally want to avoid the use of `<br>` these days. You can try wrapping it in a header tag if that's what you're looking for -- look at http://getbootstrap.com/css/#type-headings . In this case, the Bootstrap framework is being used, so one rule of thumb is to check out the documentation to achieve what you want to do (or google stackoverflow for it if you're trying to do something really uncommon). In most cases, someone has already solved your problem in an elegant way and you just need to sniff out and find the right solution (: